vsftpd/SOURCES/vsftpd.sysconfig

```diff
@@ -0,0 +1,5 @@
+# BINARY=""
```

Comment:
You should not to use commented variables in sysconfig file. In your init script, you check this variable for empty value.